scene/animation/animation_blend_tree.cpp

```diff
@@ -1328,6 +1328,24 @@ void AnimationNodeBlendTree::get_node_connections(List<NodeConnection> *r_connec
 			}
 		}
 	}
+	// Keep node connections sorted for deterministic scene file output.
+	struct Comparator {
+		bool operator()(const NodeConnection &a, const NodeConnection &b) const {
```

Comment:
```suggestion
		bool operator()(const NodeConnection &p_a, const NodeConnection &p_b) const {
```
We prefix arguments with `p_`